### PR TITLE
feat: openapi specs for add_approvers_to_report_api

### PR DESCRIPTION
**Pull request by sourav014**

### CODE REVIEW

File: src/admin/paths/admin@reports@add_approvers.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        schema:
+          type: object
+          additionalProperties: False
+          required:
+            - data
+          properties:
+            data:
+              $ref: ../../components/schemas/report.yaml#/add_approvers_in
+  responses:
+    '200':
```
Comment (kartikeyrajvaidya) [inline src/admin/paths/admin@reports@add_approvers.yaml:21]: remove approver is not bulk
so why the response is simple 200
it should ideally be report object

File: src/components/schemas/report.yaml [hunk 2]
```diff
@@ -488,6 +488,25 @@ report_action_with_expenses_ids_to_eject:
...
+  type: object
+  additionalProperties: False
+  required:
+    - id
+    - approver_emails
+  properties:
+    id:
+      type: string
+      description: |
+        id of report, for which approvers need to be added
```
Comment (kartikeyrajvaidya) [inline src/components/schemas/report.yaml:501]: `id of report, for which approvers need to be added` --> `id of the report, to which approvers need to be added`

File: src/admin/paths/admin@reports@add_approvers.yaml (hunk 1 above)
Comment (kartikeyrajvaidya) [reply src/admin/paths/admin@reports@add_approvers.yaml:21]: arey sir khudka bhi lagao na, galti ho sakti hai

**Review by kartikeyrajvaidya — APPROVED** [review]

**Comment by sourav014** [comment]
Screenshot 2023-07-05 at 11 32 58 AM

**Review by KirtiGautam — APPROVED** [review]
LGTM 🍏

**Pull request merged**